components/profileName/profileName.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Names of Folders and of Components starts from capital letter.
So folder `profileName` and component 'profileName' should start from capital letter. It goes for aboutMe folder and file too.